wdl/wdlParser.py

```diff
@@ -231,6 +232,53 @@ def module():
 
     return mod_type
 
+
+# -----------------------------------------------------------------------------
+# @fn     lvds
+# @brief  rules for the LVDSL keyword
+# @param  string slotNumber
+# @param  int type
+# @return none, appends to global variable "lvdsOutput"
+# -----------------------------------------------------------------------------
+def lvds(slot_number):
+    """
+    These are the rules for the LVDSL keyword, encountered while parsing
+    the SLOT command for the modules (.mod) file. Required format is
+    LVDSL # "label";
+
+    where # is any number
+    where "label" is a string
+    """
+    global token
+    global lvdsOutput
+
+    lvds_chan = None
+
+    if found(NUMBER):
+        lvds_chan = token.cargo
```

Comment:
how about `lvds_chan: int = token.cargo` and dispense with the `int()` later. Should be supported any time after python 3.8 I believe